x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_router.tsx

```diff
@@ -131,7 +141,7 @@ export const EngineRouter: React.FC = () => {
       )}
       {canViewEngineApiLogs && (
         <Route path={ENGINE_API_LOGS_PATH}>
-          <ApiLogs />
+          <ApiLogs />master
```

Comment:
Fix bad merge conflict (I suck at GitHub's UI apparently)
```suggestion
          <ApiLogs />
```